integration/map.go

```diff
@@ -116,9 +116,34 @@ func RunMapIntegration(ctx context.Context, mapID int64, client trillian.Trillia
 			}
 			leafHash := h.HashLeaf(leaf.LeafValue)
 			if err := merkle.VerifyMapInclusionProof(leaf.Index, leafHash, latestRoot.RootHash, incl.Inclusion, h); err != nil {
-				return fmt.Errorf("inclusion proof failed to verify for key %s: %v", incl.Leaf.Index, err)
+				return fmt.Errorf("VerifyMapInclusionProof(%x): %v", leaf.Index, err)
```

Comment:
Errors should be lower case in most cases, but this is a proper noun and I'm not sure it makes sense in this case.


The purpose of lower case error messages is to make messages of the form `log.Printf("Reading %s: %v", filename, err)` look sane.  In the case of proper nouns (the function name in this case), the lower case requirement doesn't make sense, and there's an exception for it in the [style guide](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)